invenio_app_ils/records_relations/views.py

```diff
@@ -244,11 +336,18 @@ def create(payload):
             records_to_index.append(first)
             records_to_index.append(second)
 
-            # if the record is the modified, return the modified version
-            if (
-                modified.pid == record.pid
-                and modified._pid_type == record._pid_type
-            ):
+            def is_modified(x, r):
+                return (x.pid == r.pid and x._pid_type == r._pid_type)
+
+            # NOTE: modified can be a record or a list of records, if one
+            # matches our record return the modified one.
+            if isinstance(modified, list):
```

Comment:
to keep it maybe more readable, why not:
```
_modified = modified if isinstance(modified, list) or [modified]
```

Reply:
Perfect, but all these inconsistencies will go away once we split the endpoints.